src/index.html

```diff
@@ -6,8 +6,49 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <div class="card" data-qa="card">
+        <img 
+          class="img" 
+          src="./images/imac.jpeg" 
+          alt="Apple Mac"
+        >
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="reviews">Reviews: 5</p>
+        </div>
+
+
+        <p class="price"> Price <span class="price__value">$2,199</span></p>
```

Comment:
You don't need to create separate blocks for anything other than card itself and stars. ([Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) p6)

Reply:
Not fixed. It should be an element of card